draft-ccc-wimse-twi-extensions.md

```diff
@@ -221,7 +221,10 @@ Executing the Workload inside a Trusted Execution Environment can lower the risk
 
 # Security Considerations
 
-<cref>TODO</cref>
+Maintaining security guarantees of Confidential Computing within the WIMSE reference architecture calls for adding the extensions specified in this draft.
+Specifically, strong binding between the credential and the underlying hardware platform, utilizing proof-of-possession of a workload credential backed by a key that is safeguarded against disclosure via data-in-use protections offered by Confidential Computing is a must-have.
+Additionally, invesion of the trust relationship between the workload and the hosting environment (where the workload cannot trust the hosting environment to attest it for purposes of obtaining a credential and must instead perform its own attestation) is another critical extension dictated by Confidential Computing.
+Finally, treating the pre-existing unique credential ID as a "hook" by which a Relying Party might discover information about the Workload's Provenance promises to create additional deeper access check capabilities for Relying Parties in situations such as dealing with high-risk transactions.
```

Comment:
I do not know what to unpack from "credential ID … hook … promises … deeper access check capabilities".

Reply:
I mean to say that the unique credential ID can serve as a key into the credential issuance store somewhere from where that information could be pulled.